### ISSUE TITLE
Implement "curtain plots" in Spatial -> 3D view

### ISSUE THREAD

**Issue by GoogleCodeExporter** [stoqs/stoqs#58]
```

1. Check Plot Data for a Measured Parameter for some trajectory data (e.g. AUV 
or Glider)
2. Check the Show Data box in the Spatial -> 3D panel (it will say <parameter> 
from <platform> after step 1 is done)
3. A single pixel line colored with the parameter values is show in the 3D view

We would like to optionally have interpolated values rendered in an image 
that's shown along the vehicle's section. A radio button could be added to show 
data as "section" or "lines" (or something like that). 

The implementation requires performing the gridding and color filled plotting, 
similar to the "contour" or "scatter" options in the Temporal -> Depth panel. 
This image would then be texture mapped in X3D to the geospatial coordinates of 
the vertical section occupied by the vehicle.


```

Original issue reported on code.google.com by `MBARIm...@gmail.com` on 5 Jan 2015 at 5:24

**Comment by MBARIMike** [comment]
Superseded by https://github.com/stoqs/stoqs/issues/993
